Documentation/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting started
+
+## Building the dex binary
+
+Dex requires a Go installation and a GOPATH configured. For setting up a Go workspace, refer to the [official documentation][go-setup]. Clone it down the correct place, and simply type `make` to compile the dex binary.
+
+```
+git clone https://github.com/coreos/dex.git $GOPATH/src/github.com/coreos/dex
+cd $GOPATH/src/github.com/coreos/dex
+make
+```
+
+A containerized version of this document using [rkt][rkt-repo] or Docker will be added later.
```

Comment:
This is confusing. Can you elaborate as to what that means or remove/comment it out?